torch/_inductor/ordered_set.py

```diff
@@ -0,0 +1,129 @@
+from collections.abc import MutableSet, Set as AbstractSet
+from typing import Any, cast, Iterable, Iterator, Optional, TypeVar
+
+T = TypeVar("T")
+T_co = TypeVar("T_co", covariant=True)
+
+__all__ = ["OrderedSet"]
+
+
+class OrderedSet(MutableSet[T]):
+    """
+    Insertion ordered set, similar to OrderedDict.
+    """
+
+    def __init__(self, iterable: Optional[Iterable[T]] = None):
+        self._dict: dict[T, None] = {}
+        if iterable is not None:
+            self.update(iterable)
+
+    #
+    # Required overriden abstract methods
+    #
+
+    def __contains__(self, elem: Any) -> bool:
```

Comment:
> You're probably right but there are 930 uses of `: Any` in torch and 141 uses of `: object`. I don't like really forking the idiom in two :/
> 
> Will update tho

Just because there are 930 incorrect uses of `: Any` doesn't make it "correct"...

To be clear - we're not just talking about two names for the same thing - `Any` and `object` are different and use of `Any` is almost always wrong (because it turns off typechecking). In this case it's moot because we're passing an `Any` into an `object` on an interface that can never change so the end effect is the same - but it's still "wrong" because if the thing we were passing it into were made more specific (it could never happen but imagine if python changed `__contains__` to `elem: T`) then mypy would be no help in finding those mistakes.